gradle/libs.versions.toml

```diff
@@ -147,4 +151,5 @@ slf4jNop = "org.slf4j:slf4j-nop:2.0.16"
 tikxml-htmlEscape = { module = "com.tickaroo.tikxml:converter-htmlescape", version = "0.8.15" }
 truth = "com.google.truth:truth:1.4.4"
 xmlutil-core = { module = "io.github.pdvrieze.xmlutil:core", version.ref = "xmlutil" }
-xmlutil-serialization = { module = "io.github.pdvrieze.xmlutil:serialization", version.ref = "xmlutil" }
\ No newline at end of file
+xmlutil-serialization = { module = "io.github.pdvrieze.xmlutil:serialization", version.ref = "xmlutil" }
+junit-junit = { group = "junit", name = "junit", version.ref = "junit" }
```

Comment:
We already have this dependency above :)

Reply:
😅 removed it: 60903cfba4573cb758410c4e6499205b75d6acdf thanks!